### PR TITLE
Brings back the borer event and some tweaks

### PR DESCRIPTION
**Pull request by Super3222**
Alright:
- Makes punishing victims with blindness actually blind them.
- Ports the borer icon from our old repo
- earliest_time was boosted from 20 -> 30 minutes

~~Also if there's a reason why @oisin100 didn't do this earlier, than please pause this PR until he gets back.~~

:cl: Super3222
rscadd: BORERS ARE BACK! WHOO These are alien slugs that crawl through your ear canal to reach your brain. They spawn into the game through their event again.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
could you also add borers to xenobio and give them a slightly easier to get reaction since strange reagent is annoying as all hell to get

**Comment by Nirnael** [comment]
> could you also add borers to xenobio and give them a slightly easier to get reaction since strange reagent is annoying as all hell to get

Oh so that's why he was on the watchlist to not give him borers.

**Comment by ShadowDeath6** [comment]
:'^)

to be fair omnizine and holy water are annoying to get if there's a chef / chaplain respectively because they whore all their items

File: code/modules/mob/living/simple_animal/borer/borer_event.dm [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/datum/round_event_control/borer
+	name = "Borer"
+	typepath = /datum/round_event/borer
+	weight = 15
+	max_occurrences = 1
+
+	earliest_start = 36000 // an entire hour
```
Comment (ShadowDeath6) [inline code/modules/mob/living/simple_animal/borer/borer_event.dm]: should be 45 mins, they're less of a problem than swarmers are and swarmers are 30 minutes

Comment (Super3222) [inline code/modules/mob/living/simple_animal/borer/borer_event.dm]: I'd figure with them being a neutral role.

Comment (ShadowDeath6) [inline code/modules/mob/living/simple_animal/borer/borer_event.dm]: also this shares the same start time as ninja, which is by far the most powerful role anyone can get and can solo kill literally any other antag with ease

Comment (Super3222) [inline code/modules/mob/living/simple_animal/borer/borer_event.dm]: Sounds like it

**Comment by ShadowDeath6** [comment]
:heart:

File: code/modules/mob/living/simple_animal/borer/borer_event.dm [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	if(spawned)
+		priority_announce("Unidentified lifesigns detected coming aboard [station_name()]. Secure any exterior access, including ducting and ventilation.", "Lifesign Alert", 'sound/AI/aliens.ogg') //Borers seem like normal xenomorphs.
+
+
+/datum/round_event/borer/start()
+
+	var/list/vents = list()
+	for(var/obj/machinery/atmospherics/components/unary/vent_pump/temp_vent in machines)
+		if(qdeleted(temp_vent))
+			message_admins("DEAD!")
```
Comment (X-TheDark) [inline code/modules/mob/living/simple_animal/borer/borer_event.dm]: This and other debug messages shouldn't probably be left here.

File: code/modules/mob/living/simple_animal/borer/borer_event.dm [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+			continue
+		if(temp_vent.loc.z == ZLEVEL_STATION && !temp_vent.welded)
+			message_admins("GOT A VENT!")
```
Comment (X-TheDark) [inline code/modules/mob/living/simple_animal/borer/borer_event.dm]: .

**Comment by ShadowDeath6** [comment]
> Be on the lookout for borers that may be crawling through the stations vents, you may never what hit you until they pop out!

you may never what hit you, super

**Comment by ShadowDeath6** [comment]
might want to not have the last 2 cl entries, health analyzers already tell to take sugar, and the other one is just RP fluff

**Comment by Super3222** [comment]
This PR re-introduces borers for the first time in the game. Fluff is good

**Comment by ShadowDeath6** [comment]
the CL is supposed to talk about features, not RP elements. People should know that borers can have different dispositions and reminding them of such just takes up space in the changelog when you could be talking about game features. A better CL entry would be saying that they can be detected by health analyzers which also tell them their weakness.

**Comment by Super3222** [comment]
:cl: ShadowDeath6
rscdel: FUN
/:cl:

**Comment by ShadowDeath6** [comment]
:cl: borer
rscadd: borer
/:borer:

File: code/modules/mob/living/simple_animal/borer/borer_event.dm [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+		var/client/C = pick(candidates)
+		if(!C)
+			return kill()
+
+		candidates -= C
+		var/mob/living/simple_animal/borer/borer = new(vent.loc)
+		borer.transfer_personality(C)
+		spawned = 1
+		spawncount--
+		log_game("[borer]/([borer.ckey]) was spawned as a cortical borer.")
```
Comment (oisin100) [inline code/modules/mob/living/simple_animal/borer/borer_event.dm]: We should probably notify the admins if a borer was successfully spawned and who was spawned as a borer.

**Pull request merged**